asn1/InterledgerProtocol.asn

```diff
@@ -27,4 +28,21 @@ InterledgerProtocolPaymentMessage ::= SEQUENCE {
     }
 }
 
+InterledgerProtocolError ::= SEQUENCE {
+    -- Standardized error code
+    code IA5String (SIZE (3)),
+    -- Corresponding error name
+    codeName IA5String,
```

Comment:
If this is being transmitted in binary what's the point of including the name? You need a parser anyway so that parser might as well give you the human-readable name